### ISSUE TITLE
2.23.2, last working version, the 2.25 it should have been less than zero minutes on the addon

### ISSUE THREAD

**Issue by gabri25ele** [SmartRSS/Smart-RSS#246]
2.23.2, last working version how said here:

https://github.com/SmartRSS/Smart-RSS/issues/222#issuecomment-1454667007

and instead was abandoned? 
at least 2.23.2 was put back as the latest downloadable version. since the extensions page always reminds me that there's an update.

**Comment by Lichtenshtein** [comment]
> 2.23.2

Yeah, I've gone back to that version every time after trying to upgrade. This version doesn't have the problems with infinitely updating feeds and just non-updating feeds. 

Chromium Kernel: 118.0.5993.35. Installed Smart RSS 2.25.0 unzipped. Errors on the extension page (chrome://extensions/?errors=lommkncdgjoekohhcoecblkjbgkfkbab):

```
> Uncaught TypeError: Cannot read properties of undefined (reading 'get')
> rss.html
> scripts/app/views/ItemView.js:64

> ResizeObserver loop completed with undelivered notifications.
> rss.html
> rss.html:0 

> Uncaught Error: A "url" property or function must be specified
> index.html
> scripts/libs/backbone.min.js:1

> Uncaught TypeError: Cannot read properties of null (reading 'focus')
> rss.html#
> scripts/app/staticdb/actions.js:391

> ResizeObserver loop completed with undelivered notifications.
> rss.html#
> rss.html#:0

> Uncaught TypeError: Cannot read properties of null (reading 'focus')
> rss.html
> scripts/app/staticdb/actions.js:391 
```
![30-10-2023-102315](https://github.com/SmartRSS/Smart-RSS/assets/13855295/a264f721-5c03-45f6-a923-2c0a4c1dc7b0)

After the first run, the feeds successfully refresh 1 time and seemingly no more after. Update set for 5 minutes. If you turn off and turn on the extension the feeds are updated 1 time and then again the same issue.
